panther_controller/launch/controller.launch.py

```diff
@@ -34,31 +34,15 @@
 
 
 def generate_launch_description():
-    panther_version = LaunchConfiguration("panther_version")
-    declare_panther_version_arg = DeclareLaunchArgument(
-        "panther_version",
-    )
-
-    use_sim = LaunchConfiguration("use_sim")
-    declare_use_sim_arg = DeclareLaunchArgument(
-        "use_sim",
-        default_value="False",
-        description="Whether simulation is used",
-    )
-
-    wheel_config_path = LaunchConfiguration("wheel_config_path")
-    declare_wheel_config_path_arg = DeclareLaunchArgument(
-        "wheel_config_path",
-        description="Path to wheel configuration file.",
-    )
-
+    wheel_type = LaunchConfiguration("wheel_type")
```

Comment:
APPLIES TO ALL

Sorting this file vertically by the object's context we should better place the `LaunchConfiguration` instantiation close the the corresponding argument declaration.